tests/test_iam.py

```diff
@@ -766,6 +766,120 @@ def test_iam_user_access_key_multi(self):
         self.assertEqual(
             resources[0]['c7n:matched-keys'][0]['c7n:match-type'], 'access')
 
+    def test_iam_user_ssh_key_filter(self):
+        factory = self.replay_flight_data('test_iam_user_ssh_key_filter')
+        p = self.load_policy({
+            'name': 'iam-user-ssh-keys',
+            'resource': 'iam-user',
+            'filters': [
+                {'type': 'ssh-key', 'key': 'Status', 'value': 'Active'},
+            ]},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 2)
+        self.assertEqual(len(resources[0]['c7n:matched-ssh-keys']), 1)
+
+    def test_iam_user_ssh_key_age_filter(self):
+        factory = self.replay_flight_data('test_iam_user_ssh_key_age_filter')
+        p = self.load_policy({
+            'name': 'iam-user-old-ssh-keys',
+            'resource': 'iam-user',
+            'filters': [
+                {'type': 'ssh-key', 'key': 'Status', 'value': 'Active'},
+                {'type': 'ssh-key', 'key': 'UploadDate',
+                 'value_type': 'age', 'value': 90, 'op': 'greater-than'},
+            ]},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
+        self.assertEqual(len(resources[0]['c7n:matched-ssh-keys']), 1)
+
+    def test_iam_user_disable_ssh_keys(self):
+        factory = self.replay_flight_data('test_iam_user_disable_ssh_keys')
+        user_name = 'test2'
+        p = self.load_policy({
+            'name': 'iam-user-ssh-keys',
+            'resource': 'iam-user',
```

Comment:
this one would benefit from source: config to avoid the extranous api calls on the the other two users.